Mathlib/Order/InitialSeg.lean

```diff
@@ -50,12 +56,13 @@ structure InitialSeg {α β : Type*} (r : α → α → Prop) (s : β → β →
   /-- The order embedding is an initial segment -/
   mem_range_of_rel' : ∀ a b, s b (toRelEmbedding a) → b ∈ Set.range toRelEmbedding
 
--- Porting note: Deleted `scoped[InitialSeg]`
-/-- If `r` is a relation on `α` and `s` in a relation on `β`, then `f : r ≼i s` is an order
-embedding whose range is an initial segment. That is, whenever `b < f a` in `β` then `b` is in the
-range of `f`. -/
+-- Porting note: deleted `scoped[InitialSeg]`
```

Comment:
```suggestion
-- Porting note: Deleted `scoped[InitialSeg]`
```
very minor but I think this might make git assign the blame for this line correctly